client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -491,13 +491,19 @@ class EmailProvidersComparison extends React.Component {
 	}
 
 	handleBack = () => {
-		const { selectedSite } = this.props;
+		const { selectedSite, backPath } = this.props;
```

Comment:
Non-blocking nit: Here and below on line 505, it's easier to scan the deconstructed variable names if they are in alphabetical order.
```suggestion
		const { backPath, selectedSite } = this.props;
```